code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -518,6 +518,8 @@
 		return
 	atmos_spawn_air("TEMP=1000")
 
+/obj/machinery/smartfridge/drying_rack/CtrlClick(mob/user)
+	toggle_drying(FALSE)
```

Comment:
```suggestion
/obj/machinery/smartfridge/drying_rack/CtrlClick(mob/user)
	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
		toggle_drying(FALSE)
```
Going along assuming that you meant for silicons not to use this further away. Also this check makes carbons not have mind powers